clusters/base/falco-sci.yaml

```diff
@@ -0,0 +1,536 @@
+            "uid": "prometheus"
+          },
+          "description": "",
+          "gridPos": {
+            "h": 6,
+            "w": 5,
+            "x": 15,
+            "y": 12
+          },
+          "id": 3,
+          "options": {
+            "code": {
+              "language": "plaintext",
+              "showLineNumbers": false,
+              "showMiniMap": false
+            },
+            "content": "TODO: Add info on functional unit of Falco",
+            "mode": "markdown"
+          },
+          "pluginVersion": "10.2.3",
+          "title": "Functional Unit (U) - Falco",
+          "type": "text"
+        },
+        {
+          "datasource": {
+            "type": "prometheus",
+            "uid": "prometheus"
+          },
+          "description": "",
+          "fieldConfig": {
+            "defaults": {
+              "color": {
+                "mode": "thresholds"
+              },
+              "mappings": [],
+              "thresholds": {
+                "mode": "absolute",
+                "steps": [
+                  {
+                    "color": "green",
+                    "value": null
+                  },
+                  {
+                    "color": "red",
+                    "value": 80
+                  }
+                ]
+              }
+            },
+            "overrides": []
+          },
+          "gridPos": {
+            "h": 6,
+            "w": 4,
+            "x": 20,
+            "y": 12
+          },
+          "id": 7,
+          "options": {
+            "colorMode": "value",
+            "graphMode": "area",
+            "justifyMode": "auto",
+            "orientation": "auto",
+            "reduceOptions": {
+              "calcs": [
+                "lastNotNull"
+              ],
+              "fields": "",
+              "values": false
+            },
+            "textMode": "auto",
+            "wideLayout": true
+          },
+          "pluginVersion": "10.2.3",
+          "targets": [
+            {
+              "datasource": {
+                "type": "prometheus",
+                "uid": "prometheus"
+              },
+              "editorMode": "code",
+              "expr": "sum(increase(\n         (kepler_container_joules_total{container_name=\"falco\"}[15m:1m])\n      )\n   ) * $watt_per_second_to_kWh * $emissions_france\n# add M factor when available",
+              "instant": false,
+              "legendFormat": "__auto",
+              "range": true,
+              "refId": "A"
+            }
+          ],
+          "title": "Carbon Intensity (gCO2eq) in last 15 minutes",
+          "type": "stat"
+        }
+      ],
+      "refresh": "",
+      "schemaVersion": 39,
+      "tags": [],
+      "templating": {
+        "list": [
+          {
+            "description": "emissions_intensity_gco2_per_kwh - https://github.com/thegreenwebfoundation/co2.js/blob/main/data/output/average-intensities.json#L422-L427",
```

Comment:
Nice 👍  it would be good to link to the CO2.js docs in our docs. 

Maybe we should also link to Ember Climate since they publish the carbon intensity data?
https://ember-climate.org/